frontend/app/routes/protected/person-case/current-name.tsx

```diff
@@ -154,14 +154,18 @@ export async function action({ context, request }: Route.ActionArgs) {
         ]) satisfies v.GenericSchema<CurrentNameSessionData>;
 
         const input = {
-          preferredSameAsDocumentName: formData.get('same-name') === REQUIRE_OPTIONS.yes,
+          preferredSameAsDocumentName: formData.get('same-name')
+            ? formData.get('same-name') === REQUIRE_OPTIONS.yes
+            : undefined,
           firstName: String(formData.get('first-name')),
           middleName: trimToUndefined(formData.get('middle-name') as string),
           lastName: String(formData.get('last-name')),
-          supportingDocumentsRequired: formData.get('docs-required') === REQUIRE_OPTIONS.yes,
+          supportingDocumentsRequired: formData.get('docs-required')
+            ? formData.get('docs-required') === REQUIRE_OPTIONS.yes
+            : undefined,
           supportingDocumentTypes: formData.getAll('doc-type') as string[],
         } satisfies OmitStrict<Partial<v.InferInput<typeof schema>>, 'preferredSameAsDocumentName'> & {
-          preferredSameAsDocumentName: boolean;
+          preferredSameAsDocumentName: boolean | undefined;
```

Comment:
I will have to review the validation schemas. I don`t think it`s needed.

Reply:
![image](https://github.com/user-attachments/assets/4e591f29-9f30-42f6-a168-e2a74df96f2b)

Reply:
French keyboard layout was `ON`. lol